src/App.tsx

```diff
@@ -20,9 +21,9 @@ export const App: React.FC<{}> = () => (
             <Route exact path="/">
               <HomePage />
             </Route>
-            {ITProfessions.map((r) => (
-              <Route exact path={`/${r}`} key={r}>
-                <p>Works {r}</p>
+            {ITProfessions.map((profession) => (
+              <Route exact path={`/details/${profession.name}`} key={profession.name}>
+                <DetailPage profession={profession.name} description={profession.description} />
```

Comment:
We will need also other information inside details component (for example icon or color for each path), so I would suggest to pass the whole profession object as props.

Reply:
Done